backend/api/Controllers/RobotController.cs

```diff
@@ -94,4 +86,80 @@ public async Task<ActionResult<Robot>> PostRobot([FromBody] Robot robot)
             throw;
         }
     }
+
+    /// <summary>
+    /// Start a mission for a given robot
+    /// </summary>
+    /// <remarks>
+    /// <para> This query starts a mission for a given robot and creates a report </para>
+    /// </remarks>
+    [HttpPost]
+    [Route("{robotId}/start/{missionId}")]
+    [ProducesResponseType(typeof(Report), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<Report>> StartMission([FromRoute] string robotId, [FromRoute] string missionId)
+    {
+        try
+        {
+            var robot = await _robotService.Read(robotId);
+            if (robot == null)
+            {
+                return NotFound($"Could not find robot with robot id {robotId}");
+            }
+            var report = await _robotService.StartMission(robot, missionId);
+            return Ok(report);
+        }
+        catch (MissionException me)
+        {
+            _logger.LogError(me, "Unable to start mission");
+            return NotFound(missionId);
+        }
```

Comment:
```suggestion
        catch (MissionException e)
        {
            _logger.LogError(e, "Unable to start mission");
            return NotFound(missionId);
        }
```
Can I suggest sticking to `e` for `exception` or `error`? Very readable when it's uniform across everything and the `me` variable name threw me off a bit.

Reply:
Fixed